benchmark/benchmarks.jl

```diff
@@ -3,6 +3,7 @@ using LinearAlgebra
 using Meshes
 using MeshIntegrals
 using Unitful
+using Enzyme
```

Comment:
```suggestion
import Enzyme
```
There is a namespace error with `jacobian`